include/diaspora.php

```diff
@@ -701,6 +704,27 @@ public static function handle_from_contact($contact_id, $gcontact_id = 0) {
 	}
 
 	/**
+	 * @brief get a url (scheme://domain.tld/u/user) from a given Diaspora*
+	 * fcontact guid
+	 *
+	 * @param mixed $fcontact_guid Hexadecimal string guid
+	 *
+	 * @return string the contact url or null
+	 */
+	public static function url_from_fcontact_guid($fcontact_guid) {
```

Comment:
We should only use public functions when we intent to use them outside the class. This helps very much when you are reorganizing the classes or you maybe need to rearrange function parameters.

This is comparable with the rule to only use global variables when you really are in need of them.